extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -39,4 +72,145 @@ function renderVideoAccordian(document, videoObjects) {
     document.getElementById("nav-videos").appendChild(accordianDiv);
 }
 
-export { renderVideoAccordian };
+function renderActiveFeedback(video, response) {
+    let color= "#5959e6";
+    let activeChart = document.getElementById(`activeFeedback_${video.videoID}`).getContext("2d");
+    response.active_questions.sort((a, b) => {
+        return a.timestamp - b.timestamp;
+    });
+    let labels = response.active_questions.flatMap((question) => {
+        if (question.timestamp !== null) {
```

Comment:
test truthy value with `if (question.timestamp !== null)`

Reply:
That's what I'm doing.

Reply:
I cannot do that. 0 is not truthy and we have data at timestamp zero.